nh_observations/frequencies.py

```diff
@@ -44,18 +44,13 @@
     ONE_WEEK: EVERY_WEEK
 }
 
-# To summarise the mappings below, frequencies after refusals are capped at
-# 24 hours.
-#
-# Could have just made a simple function but decided to write them out long
-# hand as they may change in the future. Return the tuple with the label rather
-# than just the time so it can be useful in more scenarios.
-PATIENT_REFUSAL_ADJUSTMENTS = {
+# Frequency tuples organised by risk and frequency in minutes.
```

Comment:
Add tech debt to make this a model and only have particular parts in their relevant modules.